src/lib/rpc.ts

```diff
@@ -10,7 +10,7 @@ import { makeFollower, makeLeader } from '@agoric/casting';
 
 export const networkConfigUrl = (agoricNetName: string) => {
   if (agoricNetName === 'local') {
-    return 'http://0.0.0.0:3000/wallet/network-config';
+    return 'https://wallet.agoric.app/wallet/network-config';
```

Comment:
I don't follow why these RPC path changes are necessary. They really should be in separate commits than the UI changes.

Reply:
the RPC paths seems like a different category of fix than UI tweaks to me. if we had downstream consumers of the we should convey the change through the commit history. not a blocker for now but something to keep in mind around communicating changes.

Reply:
Sure makes sense, I guess since it's not feature/bug. but rather development related.